File: TS-ECMA/Scripts/ex0/automovil.ts
class Automovil{
    marca:string;
    private modelo?:string;

    constructor(marca, modelo){
        this.marca = marca;
        this.modelo = modelo;
    }

    avanzar(){
        console.log('Estoy avanzando')
    }
}
let auto = new Automovil('Mazda',2021);
console.log(auto);

File: intro-java-kotlin/Scripts/kotlin-codelab/src/main/kotlin/io/academik/Automovil.kt
package io.academik

class Automovil(marca:String = "Totoya", modelo:Int = 2020, color:String = "Negro") :
    Vehiculo(marca,modelo,color) {

    override fun avanzar() {
        println("Soy el auto ${this.marca} color ${this.color} del año ${this.modelo} y estoy avanzando")
    }
}
